### ISSUE TITLE
tensorflow 버전의 sample 파일이 필요합니다.

### ISSUE THREAD

**Issue by vswnsrbvs** [AI-GrandChallenge/round-1#96]
sample 파일이 pytorch 기반이라 수정하여 작성하는데 어려움이 있습니다.

모델을 만들어 놓고도 올리지 못하는 상황입니다.

tensorflow버전의 sample 파일을 공개 해주셨으면 좋겠습니다.

**Comment by puluke4** [comment]
네~ 저도 같은 의견입니다.  알고리즘 및 코딩실력을 측정하는 것인데 특정 프로그램언어가 아니라 다른 프로그램언어로 된 것도 제공되어야 하는 것이 타당할 것 같습니다.

**Comment by appleMango7822** [comment]
저 또한 같은 의견입니다. 트랙 별 tensorflow 버전의 sample 파일은 공개할 예정이 없는 것인지요?

**Comment by lifefeel** [comment]
기존 예제로 된 save, load 내용은 그대로 두고, infer 부분에서 직접 구현한 tensorflow 모델을 불러와서 돌도록 하면 됩니다. 첨엔 load, save 내용을 지웠다가 안되어서 그렇게 했습니다.
